docs/content/quickstart.md

```diff
@@ -51,9 +47,25 @@ $ spin up --follow-all
 
 When you navigate to `http://localhost:3000`, you should see the website running.
 
+## Customize Your Website
+
+We are now at the point where you can start adding new content and also customizing the website. 
+
+### Bartholomew CLI
+
+The Bartholomew Command Line Interface (CLI) is a handy tool that will help you add new content and also check and validate existing content. Please take a minute or two and install the CLI. Here are two options:
+- download the latest `bart` binary [release](https://github.com/fermyon/bartholomew/releases/), or
+- clone and install `bart` from source, using the following commands:
```

Comment:
This doesn't "install" bart, it only _builds_ it.  Pedantic though it may seem, I think it's worth being precise about that distinction.